src/styles/mobile-nav.scss

```
@@ -0,0 +1,38 @@
+.mobile-nav {
+  position: absolute;
+  right: -100vw;
+  z-index: 2;
+  padding-left: 20px;
+  padding-right: 10px;
+  display: none;
+
+  background-color: $blue;
+  height: 100vh;
+  width: 100%;
+  font-size: 22px;
+  line-height: 22px;
+  text-transform: uppercase;
+  transition: transform 0.5s ease;
```

з сортуванням треба попрацювати [за правилами](http://codeguide.academy/html-css.html#css-order)
```suggestion
  position: absolute;
  right: -100vw;
  
  display: none;
  width: 100%;
  height: 100vh;
  padding-left: 20px;
  padding-right: 10px;
  
  font-size: 22px;
  line-height: 22px;
  text-transform: uppercase;
  
  background-color: $blue;
  transition: transform 0.5s ease;
  
  z-index: 2;
```